docs/getting-started/integrations/angular.md

```diff
@@ -690,6 +690,19 @@ export class AppComponent {
 
 For advanced usage see the {@link getting-started/setup/ui-language Setting the UI language} guide.
 
+<info-box warning>
+There is a known issue related to the localization in Angular 17. Read more in the [known issues section](#known-issues) below.
+</info-box>
+
+## Known issues
+
+The `moduleResolution` option of the TypeScript configuration determines the algorithm for finding and resolving modules from `node_modules`. In Angular 17, the option is set to `node` by default. This option prevents type declaration for editor translations from being correctly loaded. To fix it, you have several options:
+
+* You can set the `moduleResolution` option to `bundler`. It is the recommended setting in TypeScript 5.0+ for applications that use a bundler. And it is a recommended way of fixing this problem. You can check other solutions for lower TypeScript versions.
```

Comment:
> You can check other solutions for lower TypeScript versions.

You mean listed below, or other solutions for `moduleResolution`? 

This line doesn't tell exactly  what `other` solutions you meant - maybe would be good to specify it? :thinking:

Reply:
Maybe. If there is confusion, I can explicitly point to these below.